money_honey_cr.py

```diff
@@ -0,0 +1,147 @@
+import json
+import sys
+import getopt
+from decimal import Decimal, InvalidOperation
+from random import randint
+
+from actual_currencies import CurrenciesChanger
+
+
+class MoneyHoney:
```

Comment:
I guess it doesn't hurt to add it there so I added it in commit [#6](https://github.com/ZlatyChlapec/MoneyHoney/commit/f69f0f6253e8de5851e05a24c577f10b1f4c90ae). As I read up it doesn't matter in Python 3. So I guess it's more just like convention?